jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannelState.java

```diff
@@ -904,8 +904,6 @@ public void sendError(int code, String message)
                 default:
                     throw new IllegalStateException(getStatusStringLocked());
             }
-            if (_outputState != OutputState.OPEN)
-                throw new IllegalStateException("Response is " + _outputState);
```

Comment:
Why was this removed?

Reply:
I would also prefer a better error message ...

``` java
            if (_outputState != OutputState.OPEN)
                throw new IllegalStateException("Response is not open (was " + _outputState + "));
```

Reply:
See lines 895 and 896 above.... it was duplicated.